storages/python/tests-python/test_mongodb-atlas_python.py

```diff
@@ -0,0 +1,41 @@
+"""Script to test the 'mongodb' DLite plugin from Python."""
```

Comment:
This script needs a little cleanup, but we can do that when we have a good and working solution.

A think it would be good to have two tests, one that uses the anonymous user which should always run, and one that require a username and password. 

We should also document a generic way to add the `user_password_file`.

Reply:
The test has been modified to take all the relevant information from environment variables, not sure if we need to document this as it is a standard technique. Our CI would then need to be updated to export these environment variables at run time.